androiddnsfix.go

```diff
@@ -1,4 +1,4 @@
-// +build androiddnsfix
+//go:build androiddnsfix
```

Comment:
What is the difference when adding `go:`

Reply:
both do the same thing, it's the different syntax for older versions (before Go version 1.16 it used `// +build` ), and both syntaxes will work. It gets converted it to new syntax with `//go:build ` if you do a `gofmt`   command